docs/run-platform-server/ingestion-filtering.mdx

```diff
@@ -3,89 +3,61 @@ title: Ingestion Filtering
+- When filter rules are changed, they only apply to existing, running ingestion processes(**live** and **historical range**). They don't trigger any retro-active filtering or back-filling of existing historical data on the database.
+  - When the filter rules are updated to include additional accounts or assets in the white-list, the related transactions from **live** ingestion will only appear in the historical database data once the filter rules have been updated using the Admin API. The same applies to **historical range** ingestion, where the new filter rules will only affect the data from the current ledger within its configured range at the time of the update.
+  - Updating the filter rules to include additional accounts or assets does not trigger automatic back-filling related to new entites in the historical database. To include prior history of newly white-listed entites in the database you can manually run a new [Historical Ingestion Range](ingestion.mdx#ingesting-historical-data) after updating the filter rules.
+  - When the filter rules are updated to remove accounts or assets previously defined on white-list, the historical data in the database will not be retroactively purged or filtered based on the updated rules. The data is stored in the history tables for the lifetime of the database or until the `HISTORY_RETENTION_COUNT` is exceeded. Once the retention limit is reached, Horizon will purge all historical data related to older ledgers, regardless of any filtering rules.
+- Filtering will not affect the performance or throughput rate of an ingestion process, it will remain consistent whether filter rules are present or not.
 
-Whitelisting can include the following supported entities:
+Filter rules define white-lists of the following supported entities:
 
 - Account id
 - Asset id (canonical)
 
-Given that all transactions related to the white listed entities are included, all historical time series data related to those transactions are saved in horizon's history db as well. For example, whitelisting an Asset will also persist all Accounts that interact with that Asset and vice versa, if an Account is whitelisted, all assets that are held by that Account will also be included.
+Given that all transactions related to the white listed entities are included, all historical time series data related to those transactions are saved in horizon's history db, including transaction itself, all operations in the transaction, and references to any ancillary entities from operations.
 
 ## Configuration:
 
-The filters and their configuration are optional features and must be enabled with horizon command line or environmental parameters:
+Filtering is enabled by default but with no filter rules, which effectively means no filtering of ingested data occurs. To start filtering ingestion:
```

Comment:
I am unsure if by "no filtering of ingested data" means that all data is filtered out (and not stored), or if it means that all data is stored? If it's the latter, considering we only have allow-lists, it seems a bit counterintuitive to me that having no allow-list rules would result in everything allowed and stored.

Reply:
updated this with more explanation on how at least one filter rule is needed to trigger filtering, if no filter rules then triggers no filtering. related pr on making this effective - https://github.com/stellar/go/pull/4955